include/exiv2/datasets.hpp

```diff
@@ -172,15 +172,12 @@ namespace Exiv2 {
         static const uint16_t Preview                = 202;
         //@}
 
-    private:
-        //! Prevent construction: not implemented.
-        IptcDataSets() {}
```

Comment:
This one was implemented. Bug or not?

Reply:
I noticed the same, but if tests are passing ... I would not worry to much about it.

Reply:
Yes, I also wondered. Probably a bug.